### PR TITLE
Add window guide support

### PR DESCRIPTION
**Pull request by elsiosanchez**
Peek 2022-06-27 14-48

### CODE REVIEW

File: src/layout/components/Navbar.vue [hunk 1]
```diff
@@ -131,6 +131,17 @@ export default {
     fieldPanel() {
       return this.$store.getters.getStoredFieldsFromProcess(this.$route.meta.uuid).filter(field => field.isMandatory || field.isShowedFromUser)
     },
+    fieldtab() {
```
Comment (EdwinBetanc0urt) [inline src/layout/components/Navbar.vue]: Change to camel case

File: src/layout/components/Navbar.vue [hunk 2]
```diff
@@ -131,6 +131,17 @@ export default {
...
+      const tab = this.$store.getters.getStoredWindow(this.$route.meta.uuid)
+      if (!this.isEmptyValue(tab)) {
+        const tabChildIndex = parseInt(this.$route.query.tabChild)
+        const tabIndex = parseInt(this.$route.query.tab)
+        const listFieldTab = tab.tabsListParent[tabIndex].fieldsList.filter(field => field.isMandatory || field.isShowedFromUser)
```
Comment (EdwinBetanc0urt) [inline src/layout/components/Navbar.vue]: change name to fieldsListParentTab

File: src/layout/components/Navbar.vue [hunk 3]
```diff
@@ -131,6 +131,17 @@ export default {
...
+        const tabsListChild = tab.tabsListChild[tabChildIndex].fieldsList.filter(field => field.isMandatory || field.isShowedFromUser)
```
Comment (EdwinBetanc0urt) [inline src/layout/components/Navbar.vue]: change to fieldsListChildTab

**Review by EdwinBetanc0urt — CHANGES_REQUESTED** [review]

File: src/layout/components/Navbar.vue (hunk 3 above)
Comment (elsiosanchez) [reply src/layout/components/Navbar.vue]: resolve

File: src/layout/components/Navbar.vue (hunk 2 above)
Comment (elsiosanchez) [reply src/layout/components/Navbar.vue]: resolve

File: src/layout/components/Navbar.vue (hunk 1 above)
Comment (elsiosanchez) [reply src/layout/components/Navbar.vue]: resolve

**Review by EdwinBetanc0urt — APPROVED** [review]

**Pull request merged**